learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -9,12 +9,144 @@
+| [Strapi](https://github.com/meilisearch/strapi-plugin-meilisearch) | ✅ | ✅ | ❌ | ❌ |
+| [Gatsby](https://github.com/meilisearch/gatsby-plugin-meilisearch) | 🔶 <br> January 2022 | ✅ | ✅ | ❌ |
+| [Firebase](https://github.com/meilisearch/firestore-meilisearch) | 🔶 <br> January 2022 | ✅ | ✅ | ❌ |
+
+#### Configure
+
+##### Document schema
+
+| -       | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| Schemaless | ✅ | ✅ | 🔶 <br>Automatic schema detection is supported but needs to be specified. | ✅ |
+| Automatic document ID detection | ✅ | ❌ | ❌ | ❌ |
+| Native document formats | `Json`, `NDJson`, `CSV` | `Json` | `Json`, `NDJson` | `Json`, `NDJson`, `CSV` |
+
+##### Relevancy
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| Typo tolerant  | ✅ | ✅ | ✅ | 🔶 <br>Need to be specified by fuzzy queries. |
+| Orderable ranking rules | ✅ | ✅ | 🔶 <br>Tie-breaking order is limited by a unique scoring rule. | ❌|
+| Custom rules | ✅ | ✅ | 🔶 <br>Limited to one default sorting rule. | 🔶 <br>Function score query
+| Query field weights | ✅ | ✅ | ✅ | ✅ |
+| Synonyms | ✅ | ✅ | ✅ | ✅ |
+| Stop words | ✅ | ✅ | ❌ | ✅ |
+| Automatic language detection | ✅ | ✅ | ❌ | ❌ |
+| All language supports | ✅ | ✅ | ❌ <br> Only space separated | ✅ |
+
+##### Security
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| API Key Management | ✅ | ✅ | ✅ | ✅ |
+| Tenant tokens & multi-tenant indexes | **Q1 2022** <br> Configurable end-user hard filters per index. | ✅  <br> Hard filters are not configurable per index for an end-user tenant key.   | ✅ <br> Hard filters are not configurable per index for an end-user tenant key. | ✅ <br> Role based |
+
+##### Search
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| Placeholder search | ✅ | ✅ | ✅ | ✅ |
+| Multi-index search | Early 2022 | ✅ | ✅ | ✅ |
+| Exact phrase search | ✅ | ✅ | ❌ | ✅ |
+| Geo search |  ✅  | ✅ | ✅ | ✅ |
+| Sort by  |  ✅  | 🔶 <br> Limited to one `sort_by` rule per index. Index have to be duplicated for each sort field and sort order. | 🔶 <br> Do not support sort on string field. | ✅ |
+| Filtering |  ✅ <br>Support complex filter queries with an SQL-like syntax.  | 🔶 <br>Does not support `OR` operation across multiple fields. | 🔶 <br>Does not support `OR` operation across multiple fields. | ✅ |
+| Facetting |  ✅ | ✅ | ✅ | ✅ |
+| Facet value searches | ❌ | ✅ | ✅ | ❌ |
+| Distinct <br><div style="color:#A9A9A9;margin-top:0.8em;">De-duplicate documents by a field value</div>| ✅ | ✅ | ✅  | ✅ |
+| Grouping <br><div style="color:#A9A9A9;margin-top:0.8em;">Bucket documents by field values</div> | ❌ | ✅ | ✅  | ✅ |
+
+##### Vizualize
+
+| - | MeiliSearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| [Visual Dashboard](https://github.com/meilisearch/mini-dashboard) | ✅ | 🔶 <br>Cloud product | 🔶 <br>Cloud product | ✅ |
```

Comment:
Trying to make this consistent with current usage across the docs.

```suggestion
| [Mini Dashboard](https://github.com/meilisearch/mini-dashboard) | ✅ | 🔶 <br> Cloud product | 🔶 <br> Cloud product | ✅ |
```

Reply:
unsure about this. @gmourier ?

Reply:
It seems ok to me! The idea here is to highlight the fact that we ship a tool that allows seeing visually what is in an instance.